react-native/push/App.js

```diff
@@ -24,16 +28,16 @@ export default class App extends Component {
 
       },
     };
-    
     let callback = (message)=>{
       console.log("You have receieved a push message." + JSON.stringify(message));
       this.setState({
         ...this.state,
         messages:[...this.state.messages, message]
       });
     };
+    
+    ups.registerMessageHandler(callback);    
```

Comment:
I believe the callback should be a `const`.
Even better, it could be inlined:
```suggestion
       ups.registerMessageHandler(message => this.setState({messages: [...this.state.messages, message] });    
```
We don't need to specify the `...this.state`: only the specified key will be changed by the setState